src/main/java/mate/academy/bookshop/repository/book/providers/PriceSpecificationProvider.java

```diff
@@ -16,11 +19,11 @@ public String getKey() {
 
     @Override
     public Specification<Book> getSpecification(String[] params) {
-        if (params == null || params.length != 2) {
+        if (params == null || params.length != EXPECTED_LENGTH) {
             throw new IllegalArgumentException("Invalid price range parameters");
         }
-        BigDecimal minPrice = BigDecimal.valueOf(Long.parseLong(params[0]));
-        BigDecimal maxPrice = BigDecimal.valueOf(Long.parseLong(params[1]));
+        BigDecimal minPrice = BigDecimal.valueOf(Long.parseLong(params[MIN_PRICE_INDEX]));
+        BigDecimal maxPrice = BigDecimal.valueOf(Long.parseLong(params[MAX_PRICE_INDEX]));
 
         return (root, query, criteriaBuilder)
                 -> criteriaBuilder.between(root.get("price"), minPrice, maxPrice);
```

Comment:
Does this method assume that the params[] will always contain valid numeric strings that can be parsed as long integers?